lib/utils.coffee

```diff
@@ -320,11 +311,11 @@ requestAsync = (options, retriesRemaining) ->
 # @param {Object} options
 #
 # @example
-# utils.requestAsync({ url: 'http://example.com' }).then (response) ->
+# utils.requestAsync()({ url: 'http://example.com' }).then (response) ->
 # 	console.log(response)
 ###
-exports.requestAsync = (options) ->
-	requestAsync(options)
+exports.requestAsync = (fetch = normalFetch) -> (options) ->
```

Comment:
Can we rename this to `getRequestAsync`, since it's now a method that returns the old `requestAsync` method?